gno.land/pkg/gnoclient/client_txs.go

```diff
@@ -33,63 +67,64 @@ type RunCfg struct {
 }
 
 // Call executes a contract call on the blockchain.
-func (c *Client) Call(cfg CallCfg) (*ctypes.ResultBroadcastTxCommit, error) {
+func (c *Client) Call(cfg BaseTxCfg, msgs ...MsgCall) (*ctypes.ResultBroadcastTxCommit, error) {
 	// Validate required client fields.
 	if err := c.validateSigner(); err != nil {
-		return nil, errors.Wrap(err, "validate signer")
+		return nil, err
 	}
 	if err := c.validateRPCClient(); err != nil {
-		return nil, errors.Wrap(err, "validate RPC client")
+		return nil, err
 	}
 
-	pkgPath := cfg.PkgPath
-	funcName := cfg.FuncName
-	args := cfg.Args
-	gasWanted := cfg.GasWanted
-	gasFee := cfg.GasFee
-	send := cfg.Send
-	sequenceNumber := cfg.SequenceNumber
-	accountNumber := cfg.AccountNumber
-	memo := cfg.Memo
-
-	// Validate config.
-	if pkgPath == "" {
-		return nil, errors.New("missing PkgPath")
+	// Validate base transaction config
+	if err := cfg.validateBaseTxConfig(); err != nil {
+		return nil, err
 	}
-	if funcName == "" {
-		return nil, errors.New("missing FuncName")
+
+	// Parse MsgCall slice
+	vmMsgs := make([]vm.MsgCall, len(msgs))
+	for _, msg := range msgs {
+		// Validate MsgCall fields
+		if err := msg.validateMsgCall(); err != nil {
+			return nil, err
+		}
+
+		// Parse send coins
+		send, err := std.ParseCoins(msg.Send)
+		if err != nil {
+			return nil, fmt.Errorf("%w", err)
```

Comment:
```suggestion
			return nil, err
```
that error format with no extra info is useless.

Reply:
[8be378e](https://github.com/gnolang/gno/pull/1565/commits/8be378eb932dec25b3f949d330446dbad7fb9def)